### DISCUSSION TITLE
No icon in Ubuntu Dock

### DISCUSSION THREAD

**Discussion by hongquan** [zed-industries/zed#16503] [Feature Requests]
Installed Zed from the tarball downloaded from GitHub Releases.
When open Zed, Ubuntu Dock show general icon with cog, instead of Zed icon:

![image](https://github.com/user-attachments/assets/0faac79e-d1f5-4a58-ac15-d85c252b74f8)

**Comment by rohitshriwas** [comment]
Can confirm same issue on Elementary OS. I installed the Flatpak version of Zed. Clicking on the pinned dock icon makes it launch a new instance with a generic icon to represent it. 

![image](https://github.com/user-attachments/assets/19015ea7-c4c1-4f5b-b6c1-153ada6ecbb5)

**Comment by hongquan** [comment]
Created PR #16504 to fix.

**Comment by notpeter** [comment]
Closing discussion in favor of issue:
- https://github.com/zed-industries/zed/issues/16694

**Comment by hongquan** [comment]
If someone comes here, the workaround is to manually rename the _zed.desktop_ file in _~/.local/share/applications_ to _dev.zed.Zed.desktop_.

**Comment by xXMariahScaryXx** [comment]
still happening to me nearly 2 years later lol

**Reply by xXMariahScaryXx** [reply]
fix is to edit /usr/share/applications/dev.zed.Zed.desktop
change `Icon=zed` to `Icon=/usr/share/icons/zed.png`
